modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/info_panel/AssetTypeInfoPanelFolderView.tsx

```diff
@@ -8,7 +8,6 @@ import React, {useState} from 'react';
 
 import {TABS} from './tab_content';
 import DetailsTabContent from './tab_content/DetailsTabContent';
-import PerformanceTabContent from './tab_content/PerformanceTabContent';
 
 const AssetTypeInfoPanelFilesView = () => {
```

Comment:
Even though this is the `AssetTypeInfoPanelFolderView`, as you can see the component's name is `AssetTypeInfoPanelFilesView`. Is this just a typo? If so, let's fix this, it was probably forgotten from the past as it's `default`-exported.